tools/run.sh

```diff
@@ -4,4 +4,4 @@
 dd if=/dev/zero of=/tmp/unetcat.sock bs=1024 count=1024
 
 sudo setcap cap_net_raw,cap_net_admin,cap_net_bind_service+eip build/inetd
-sudo ip netns exec TEST su $USER -c "build/inetd $1"
+sudo ip netns exec TEST su $USER -c "build/inetd -i $1"
```

Comment:
What is the intention?

Reply:
What if no interface is found?

Reply:
@jserv  I've used the original structure - `ether_args`  array in [src/nstack.c, line 348](https://github.com/kevinbird61/nstack/blob/master/src/nstack.c#L348), only replace `argv[1]` with `intf`(**Network Interface**) and `NULL` with `mac`(**MAC address**).

And if no specify any interface for `inetd`, it will display the error message:
`Failed to init: Operation not permitted`